cranelift/filetests/filetests/pcc/fail/simple.clif

```diff
@@ -2,14 +2,25 @@ test compile expect-fail
 set enable_pcc=true
 target aarch64
 
-;; The `points_to` annotation is not large enough here -- the
-;; 4GiB-range 32-bit offset could go out of range. PCC should catch
-;; this.
+;; The `memory` memtype is not large enough here -- the 4GiB-range
+;; 32-bit offset could go out of range. PCC should catch this.
 
 function %simple1(i64 vmctx, i32) -> i8 {
-block0(v0 ! points_to(0x8000_0000): i64, v1 ! max(32, 0xffff_ffff): i32):
+    mt0 = memory 0x8000_0000
+block0(v0 ! mem(mt0, 0): i64, v1 ! max(32, 0xffff_ffff): i32):
     v2 ! max(64, 0xffff_ffff) = uextend.i64 v1
-    v3 ! points_to(0x1) = iadd.i64 v0, v2
+    v3 ! mem(mt0, 0xffff_ffff) = iadd.i64 v0, v2
+    v4 = load.i8 checked v3
+    return v4
+}
+
+;; Check that the offset in the `mem` is validated too.
+
+function %simple1(i64 vmctx, i32) -> i8 {
```

Comment:
I don't think it matters if function names are duplicated but worth making sure

```suggestion
function %simple2(i64 vmctx, i32) -> i8 {
```

Reply:
Fixed!